tests/e2e/testdata/rapid/TestChainStateMarshalling/TestChainStateMarshalling-20240314151749-32478.fail

```diff
@@ -0,0 +1,10 @@
+# 
```

Comment:
Why is this file and the one below needed here?

Reply:
removed.